linkerd/app/src/lib.rs

```diff
@@ -144,7 +145,7 @@ impl Config {
                 outbound_metrics.clone(),
             );
 
-            let span = info_span!("outbound");
+            let span = debug_span!("outbound");
```

Comment:
similarly, I think it's worth keeping the outbound and inbound spans at INFO for errors/warnings, as well as the accept span?